### DISCUSSION TITLE
Error in Timezone visualization

### DISCUSSION THREAD

**Discussion by Andre4810** [dbeaver/dbeaver#22016] [Q&A]
Hi. When I create queries and query the DataBase connected to DBeaver, I see an incorrect time.
The real time 11.10 PM (UTC+1, Rome) is displayed as 10.10 PM (UTC).
I tried changing the timezone. The change is saved but I still see the wrong time on every date
<img width="427" alt="Screenshot 2023-11-28 124052" src="https://github.com/dbeaver/dbeaver/assets/152278807/06a94f85-dce5-42df-8769-f384154fda40">
<img width="518" alt="Screenshot 2023-11-28 124438" src="https://github.com/dbeaver/dbeaver/assets/152278807/47607c16-ce3e-42aa-9581-f1228ea889b5">

Can you help me figure out what the error is?
Thanks.

**Comment by LonwoLonwo** [comment]
Hello @Andre4810 

> The real time 11.10 PM (UTC+1, Rome) is displayed as 10.10 PM (UTC).

Displayed where specificaly?

Also, I see on your second picture that you changed the data format for the specific data source. Did you changed it also for all DBeaver in the main Preferences?

**Comment by Andre4810** [comment]
Hi.
Thanks for your answer.
The wrong time is displayed in the result of the queries.
In my database, I can see the records with the correct timezone (CET).
If I try to extract them by creating a query in DBeaver, I retrieve the same records but all with wrong timezone (UTC).

Example of result in DBeaver:
<img width="320" alt="Screenshot 2023-11-29 110909" src="https://github.com/dbeaver/dbeaver/assets/152278807/559e1864-7d9a-4e01-9a9e-17710de8ed14">

The same result but in our database and not DBeaver:
<img width="182" alt="Screenshot 2023-11-29 111425" src="https://github.com/dbeaver/dbeaver/assets/152278807/ffe388a9-772c-48b0-b435-6594e4911d5d">

You can see that DBeaver reports a different time (in UTC) than the expected time (CET):
- From 00.59 to 23.59 (from 0.59 AM to 11.59 PM)
- From 14.51 to 13.51 (from 2.51 PM to 1.51 PM)

I tried to change the timezone (UTC to CET) for all DBeaver in the "Preferences" menu but I still see the dates with the time in UTC.

**Reply by LonwoLonwo** [reply]
Could you please try to change your timezone here?

![2023-11-30 14_28_23-DBeaver Ultimate 23 3 0 - NEWTABLE](https://github.com/dbeaver/dbeaver/assets/45152336/a6a18d47-fdfe-4ab3-904e-4fec3a8a60e1)

**Comment by Andre4810** [comment]
Hi.
I tried to change the timezone in this section but I still see the times incorrectly.
<img width="450" alt="Screenshot 2023-11-30 123316" src="https://github.com/dbeaver/dbeaver/assets/152278807/0c2d1b05-1a23-45b3-ad59-08bcc03284ea">

**Reply by LonwoLonwo** [reply]
What OS do you have?
